devices/tuya.js

```diff
@@ -439,6 +439,18 @@ module.exports = [
         toZigbee: [],
         exposes: [e.battery(), e.temperature(), e.humidity(), e.battery_voltage()],
     },
+    {
+        fingerprint: [
+            {modelID: 'TS0201', manufacturerName: '_TZ3000_qaaysllp'},
+        ],
+        zigbeeModel: ['TS0201_multi_sensor'],
```

Comment:
I don't think this is needed (since identification is done on the `fingerprint` above), please remove this and check if it still works.

Reply:
It is not working.
Because Round type(Old TS0201) has temperature and humidity.
But square type(New TS0201) has add illuminate.
Otherwise, Clusters is diffierent.

Round type 
{"id":17,"type":"EndDevice","ieeeAddr":"0xec1bbdfffe302afe","nwkAddr":20996,"manufId":4098,"manufName":"_TZ2000_a476raq2","powerSource":"Battery","modelId":"TS0201","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":770,"inClusterList":[0,1,1026,1029],"outClusterList":[25],"clusters":{"genPowerCfg":{"attributes":{"batteryVoltage":27,"batteryPercentageRemaining":160}},"genBasic":{"attributes":{"65506":19,"modelId":"TS0201","manufacturerName":"_TZ2000_a476raq2","powerSource":3,"zclVersion":3,"appVersion":67,"stackVersion":0,"hwVersion":1,"dateCode":""}},"msTemperatureMeasurement":{"attributes":{"measuredValue":2680}},"msRelativeHumidity":{"attributes":{"measuredValue":6740}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":67,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1630621521942}

Square type
{"id":62,"type":"EndDevice","ieeeAddr":"0xb4e3f9fffe0c464d","nwkAddr":30653,"manufId":4098,"manufName":"_TZ3000_qaaysllp","powerSource":"Battery","modelId":"TS0201","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":262,"inClusterList":[0,1,1024,57346],"outClusterList":[25,10],"clusters":{"msIlluminanceMeasurement":{"attributes":{"61441":50,"measuredValue":23055}},"genBasic":{"attributes":{"modelId":"TS0201","manufacturerName":"_TZ3000_qaaysllp","powerSource":3,"zclVersion":3,"appVersion":68,"stackVersion":0,"hwVersion":1,"dateCode":""}},"genPowerCfg":{"attributes":{"batteryPercentageRemaining":200}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":68,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1630624473474}


New type send temperature and humidity like  
"type 'raw', cluster '57346', data '{\"data\":[8,37,10,10,208,41,40,0],\"type\":\"Buffer\"

I can not reverse packet so i didn't make new converter(only make new device with illuminate).

Reply:
You mean it is not working without `zigbeeModel: ['TS0201_multi_sensor'],`?

Reply:
I understand, I removed zigbeemodel. It is working.
but it need new converter for temperature and humidity value.

Reply:
Seems the device uses a non-standard way for this (it does not have `msTemperatureMeasurement`)

https://www.zigbee2mqtt.io/how_tos/how_to_support_new_tuya_devices.html might help

Reply:
Okay. I will do that, but I think need many time.